playbooks/cert-distribute/cert_distribute.yml

```
@@ -0,0 +1,69 @@
+---
+- name: Generate a new IBM Storage Protect server certificate by using the administrative console.
+  hosts: CERT_ADMIN_CLIENT
+  gather_facts: no
+  tasks:
+    - name: Produce a certificate label that utilizes the date time.
+      shell: "echo SP_CERT_$(date '+%b_%d_%Y_%H_%M_%S')"
+      register: "sp_server_date_time"
+
+    - name: "Print Certificate Label"
+      debug:
+        var: sp_server_date_time.stdout
+
+    - name: Create a new self-signed certificate on the IBM Storage Protect server.
+      shell: "cd /opt/tivoli/tsm/client/ba/bin/;dsmadmc -id={{ lookup('ini', 'SP_SERVER_USERNAME', file='data.ini') }} -password={{ lookup('ini', 'SP_SERVER_PASSWORD', file='data.ini') }} \"create certificate \\\"{{ sp_server_date_time.stdout }}\\\"\""
```

It feels like this could be a module in itself.

Either way, could this be changed as follows:

```suggestion
      command: 
        chdir: /opt/tivoli/tsm/client/ba/bin/
        cmd: "dsmadmc -id={{ lookup('ini', 'SP_SERVER_USERNAME', file='data.ini') }} -password={{ lookup('ini', 'SP_SERVER_PASSWORD', file='data.ini') }} \"create certificate \\\"{{ sp_server_date_time.stdout }}\\\"\""
```